pom.xml

```diff
@@ -181,7 +181,7 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
                         <showDeprecation>true</showDeprecation>
                         <showWarnings>true</showWarnings>
                         <compilerArgs>
-                            <arg>-Werror</arg>
+                            <!-- arg>-Werror</arg -->
```

Comment:
Is it because of the new deprecations within the Vavr core library?

Reply:
Yes, you are right.